apps/verticalhome/test/marionette/group_drag_expanded_test.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+var Actions = require('marionette-client').Actions;
+var assert = require('assert');
+
+marionette('Vertical - Group', function() {
+
+  var client = marionette.client(require(__dirname + '/client_options.js'));
+  var actions, home, system;
+
+  setup(function() {
+    actions = new Actions(client);
+    home = client.loader.getAppClass('verticalhome');
+    system = client.loader.getAppClass('system');
+    system.waitForStartup();
+
+    client.apps.launch(home.URL);
+    home.waitForLaunch();
+  });
+
+  test('check that dragging an expanded group collapses correctly', function() {
+    function getRect(el) {
+      return el.getBoundingClientRect();
+    }
+
+    var groupBackgroundSelector = '#icons .group .background';
```

Comment:
nit: I would opt to move this into the selectors object of verticalhome.js.

https://github.com/mozilla-b2g/gaia/blob/master/apps/verticalhome/test/marionette/lib/verticalhome.js#L31